src/components/PageBlocks/RecordListBase.vue

```diff
@@ -201,7 +205,7 @@
                   v-if="field.sortable"
                   variant="link p-0 ml-1"
                   class="d-flex align-items-center justify-content-center"
-                  @click="handleSort(field)"
+                  @click.stop="handleSort(field)"
```

Comment:
Not sure if we really need this extra call for handleSort here

Reply:
Ok, will remove it than, I am also at 50:50